content/en/docs/languages/java/api-components.md

```diff
@@ -0,0 +1,1690 @@
+(i.e. metrics and traces), the same scope should be used. See
+[instrumentation scope](/docs/concepts/instrumentation-scope/) for details.
+
+The following code snippet explores provider and scope API usage:
+
+<!-- prettier-ignore-start -->
+<?code-excerpt "src/main/java/otel/ProvidersAndScopes.java"?>
+```java
+package otel;
+
+import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.api.logs.Logger;
+import io.opentelemetry.api.logs.LoggerProvider;
+import io.opentelemetry.api.metrics.Meter;
+import io.opentelemetry.api.metrics.MeterProvider;
+import io.opentelemetry.api.trace.Tracer;
+import io.opentelemetry.api.trace.TracerProvider;
+
+public class ProvidersAndScopes {
+
+  private static final String SCOPE_NAME = "fully.qualified.name";
+  private static final String SCOPE_VERSION = "1.0.0";
+  private static final String SCOPE_SCHEMA_URL = "https://example";
+
+  public static void providersUsage(OpenTelemetry openTelemetry) {
+    // Access providers from an OpenTelemetry instance
+    TracerProvider tracerProvider = openTelemetry.getTracerProvider();
+    MeterProvider meterProvider = openTelemetry.getMeterProvider();
+    // NOTE: LoggerProvider is a special case and should only be used to bridge logs from other
+    // logging APIs / frameworks into OpenTelemetry.
+    LoggerProvider loggerProvider = openTelemetry.getLogsBridge();
+
+    // Access tracer, meter, logger from providers to record telemetry for a particular scope
+    Tracer tracer =
+        tracerProvider
+            .tracerBuilder(SCOPE_NAME)
+            .setInstrumentationVersion(SCOPE_VERSION)
+            .setSchemaUrl(SCOPE_SCHEMA_URL)
+            .build();
+    Meter meter =
+        meterProvider
+            .meterBuilder(SCOPE_NAME)
+            .setInstrumentationVersion(SCOPE_VERSION)
+            .setSchemaUrl(SCOPE_SCHEMA_URL)
+            .build();
+    Logger logger =
+        loggerProvider
+            .loggerBuilder(SCOPE_NAME)
+            .setInstrumentationVersion(SCOPE_VERSION)
+            .setSchemaUrl(SCOPE_SCHEMA_URL)
+            .build();
+
+    // ...optionally, shorthand versions are available if scope version and schemaUrl aren't
+    // available
+    tracer = tracerProvider.get(SCOPE_NAME);
+    meter = meterProvider.get(SCOPE_NAME);
+    logger = loggerProvider.get(SCOPE_NAME);
+  }
+}
+```
+<!-- prettier-ignore-end -->
+
+### Attributes
+
+[Attributes](https://www.javadoc.io/doc/io.opentelemetry/opentelemetry-api/latest/io/opentelemetry/api/common/Attributes.html)
+is a bundle of key value pairs representing the
+[standard attribute definition](/docs/specs/otel/common/#standard-attribute).
+`Attributes` are a recurring concept in the OpenTelemetry API:
+
+- [Spans](#span), span events, and span links have attributes
+- The measurements recorded to [metric instruments](#meter) have attributes
+- [LogRecords](#logrecordbuilder) have attributes
+- and more
```

Comment:
not sure if this adds anything, consider removing?

```suggestion
```

Reply:
In my head I was leaving the door open for other usages of context, so that it was clear that this isn't necessarily an exhaustive list. But let's just omit it.